docs/src/guides/run-ingest-on-terra.rst

```diff
@@ -0,0 +1,79 @@
+**************************
+Run Data Ingest on Terra
+**************************
+
+We have provided two pipelines for importing data into Terra:
+
+* GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.
+* GISAID Ingest - pull a private dataset if a user has their own API key, account, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagramed below:
+
+.. image:: ../images/terra-ingest.png
+
+
+Import ``ingest`` wdl workflow from Dockstore
```

Comment:
`ingest` isn't the exact name so I wouldn't stylize it as an inline literal. Also, inline literals in headings are rendered a bit small in the preview. This might be good enough given the existing context:

```suggestion
Import the wdl workflow from Dockstore
```